.github/workflows/release-prepare-hotfix.yml

```diff
@@ -85,9 +85,9 @@ jobs:
       - name: merge into release if present
         id: merge-release
         run: |
-          if [[ git checkout release ]]
+          if git checkout release
           then
-            if [[ git merge ${{ github.event.inputs.ref }} ]]
+            if git merge ${{ github.event.inputs.ref }}
```

Comment:
if the merge fails, wondering if `git merge --abort` would be needed?

Reply:
Good point, could be good to do at least to be safe

Reply:
Added